src/components/bulkConcepts/container/BulkConceptsPage.jsx

```diff
@@ -91,7 +83,7 @@ export class BulkConceptsPage extends Component {
 
   handleSearch = (event) => {
     event.preventDefault();
-    if (this.state.searchInput.trim()) {
+    if (this.state.searchInput.trim() !== this.state.searchInput) {
```

Comment:
@Duncanian, I think it would be best if you destructured state here.

Reply:
Okay, let me work on it.